### ISSUE TITLE
Error getting metadata

### ISSUE THREAD

**Issue by 97carmine** [jellyfin/jellyfin-plugin-bookshelf#3]
Once the library is managed, when manually indicating the search for new content, the log shows the following error

```
[2019-08-11 11:29:27.009 +02:00] [ERR] Error in "Open Packaging Format"
System.NullReferenceException: Object reference not set to an instance of an object.
   at Jellyfin.Plugin.Bookshelf.Providers.BookProviderFromOpf.ReadOpfData(MetadataResult`1 bookResult, String metaFile, CancellationToken cancellationToken)
   at Jellyfin.Plugin.Bookshelf.Providers.BookProviderFromOpf.GetMetadata(ItemInfo info, IDirectoryService directoryService, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.MetadataService`2.RefreshWithProviders(MetadataResult`1 metadata, TIdType id, MetadataRefreshOptions options, List`1 providers, ItemImageProvider imageService, CancellationToken cancellationToken)
```

**Comment by telans** [comment]
I don't encounter this, is it still an issue for you?

**Comment by 97carmine** [comment]
Yes, it still happens to me

**Comment by oddstr13** [comment]
Same issue as #14 
Fixed in #10
